script/Ciel/Day 10/10_01_CIEL10_5_2.txt

```diff
@@ -215,6 +215,7 @@ It's the voice of a woman I know...
 }
 [sha:720ed823226be254be915ab2f068d1f79cd8c386]{
 -- Page 394, Offset 28151.
+// why are we keeping these ―――, ――― now but not the other ones from earlier
```

Comment:
They should probably be replaced with appropriate onomatopoeia

Reply:
Will do. There's a bunch of these too.